src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -1458,74 +1509,115 @@ void CouplingScheme::computeCommonPlaneTimeStep(real &dt)
       //          common-plane method                          //
       ///////////////////////////////////////////////////////////
 
-      // compute delta between velocity projection of face-projected 
-      // overlap centroid and the OTHER face's face-projected overlap 
-      // centroid
-      real proj_delta_x1 = cpMgr.m_cXf1[cpID] + dt * vel_f1[0] - cpMgr.m_cXf2[cpID];
-      real proj_delta_y1 = cpMgr.m_cYf1[cpID] + dt * vel_f1[1] - cpMgr.m_cYf2[cpID];
-      real proj_delta_z1 = 0.;
-
-      real proj_delta_x2 = cpMgr.m_cXf2[cpID] + dt * vel_f2[0] - cpMgr.m_cXf1[cpID]; 
-      real proj_delta_y2 = cpMgr.m_cYf2[cpID] + dt * vel_f2[1] - cpMgr.m_cYf1[cpID];
-      real proj_delta_z2 = 0.;
-
-      // compute the dot product between each face's delta and the OTHER 
-      // face's outward unit normal. This is the magnitude of interpenetration 
-      // of one face's projected overlap-centroid in the 'thickness-direction' 
-      // of the other face (with whom in may be in contact currently, or in 
-      // a velocity projected sense).
-      real proj_delta_n_1 = proj_delta_x1 * fn2[0] + proj_delta_y1 * fn2[1];
-      real proj_delta_n_2 = proj_delta_x2 * fn1[0] + proj_delta_y2 * fn1[1];
-
-      if (dim == 3)
+      // check for case where both faces have nearly zero velocity. If so,
+      // don't compute a velocity projection dt estimate
+      real tiny_vel_proj = 1.e-8;
```

Comment:
I know you just moved this, but this dimensional constant is a bit concerning.

Reply:
Per our discussion, I removed this check. There are additional complexities to doing this correctly. By removing this check, we simply compute the velocity projected dt estimate for all interface pairs, even those with 'nearly' zero velocity. This change was made for simplicity and clarity.